doc/tutorials/biicode/table_of_content_biicode.markdown

```diff
@@ -0,0 +1,11 @@
+OpenCV with biicode dependency manager {#tutorial_table_of_content_biicode}
+============================================
+
+
+-   @subpage tutorial_biicode
+
+    *Compatibility:* \> OpenCV 2.4
+
+    *Author:* biicode
+
+    Here we learn how to use OpenCV with the simplicity of *biicode* dependency manager for C/C++.
```

Comment:
We will learn how to* use OpenCV with the simplicity of *biicode*, a dependency manager for C/C++.